cmd/up/forwards.go

```diff
@@ -105,5 +118,89 @@ func (up *upContext) sshForwards(ctx context.Context) error {
 		return fmt.Errorf("failed to add entry to your SSH config file")
 	}
 
-	return up.Forwarder.Start(up.Pod.Name, up.Dev.Namespace)
+	err := up.Forwarder.Start(up.Pod.Name, up.Dev.Namespace)
```

Comment:
This is duplicated code that could lead to bugs possibly if one is updated and not the other. Can we use a function?